### ISSUE TITLE
Unit test coverage not happening

### ISSUE THREAD

**Issue by rmsmani** [localstack/localstack#2346]
# Type of request: This is a ...

[X ] bug report
[ ] feature request

# Detailed description
Created some of the test cases  under  https://github.com/localstack/localstack/blob/master/tests/unit/test_common.py

But these test cases are not covered in the latest build 
https://coveralls.io/builds/30298765/source?filename=localstack%2Futils%2Fcommon.py

Eg: The below test case should hv covered, but the coverage didnt happen
https://github.com/localstack/localstack/blob/master/tests/unit/test_common.py#L8

Coverage didnt happen
<img width="483" alt="coverage-error" src="https://user-images.githubusercontent.com/28680638/80177767-fc2b9e00-863f-11ea-88af-9bf4cd792290.png">

## Expected behavior
The coverage should happen for the unit test cases

## Actual behavior

...

# Steps to reproduce

## Command used to start LocalStack

...

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

...

**Comment by rmsmani** [comment]
@whummer ,
What need to be done to have the test cases covered

**Comment by whummer** [comment]
Thanks for digging into this @rmsmani . This should be fixed in #2353 - if you look at the comment in that PR, the coverage is now correctly reported as `72.426%`:
```
Coverage increased (+21.6%) to 72.426% when pulling 356929a on docker-cp-coverage into 1aa5d52 on master.
```
